legacy/firmware/layout2.c

```diff
@@ -781,8 +781,12 @@ void layoutAddress(const char *address, const char *desc, bool qrcode,
     }
 
     oledInvert(0, 0, 63, 63);
-    if (side > 0 && side <= 29) {
-      int offset = 32 - side;
+    if (side > 0 && side <= 33) {
+      // For QR code V4 (33x33), still maintaining two pixels per module
+      // so that it is nicely readable. Not showing the bottom row,
+      // which is not a problem thanks to the QR code's error correction.
+      // It is a use-case for BTC taproot addresses.
+      int offset = side <= 29 ? 32 - side : 0;
```

Comment:
Did you try using -2 instead of 0? Might look nicer aligned to center and we'd lose two half-rows instead of single full row:thinking:

Reply:
I tried `-1` for the `offset`, but the issue is that it also creates `x` offset, so even the left half-row is cut-off this way. Surely we could have two offsets, one for `x` (0) and one for `y` (-1), but that would complicate the code unnecessarily IMO